tv.kodi.Kodi.json

```diff
@@ -0,0 +1,128 @@
+{
+    "app-id": "tv.kodi.Kodi",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "1.6",
```

Comment:
You should use 18.08 instead.

Reply:
No it doesn't. You just have to build the library without the applications. Then it doesn't fail.

Reply:
yes, I got this working